WikipediaUnitTests/Code/MWKTitleTests.m

```diff
@@ -48,7 +48,6 @@ - (void)testSimple {
     MWKTitle* title = [MWKTitle titleWithString:@"Simple" site:site];
     XCTAssertEqualObjects(title.text, @"Simple", @"Text form is full");
     XCTAssertNil(title.fragment, @"Fragment is nil");
-    XCTAssertEqualObjects(title.escapedFragment, @"", @"Fragment for URL is empty string");
```

Comment:
Is the big change here is that we are saying that fragments can be nil where before they would only be empty strings?